lib/db-utils.js

```diff
@@ -2,21 +2,35 @@ const dotenv = require("dotenv");
 dotenv.config();
 const { generateKey } = require("../data/dataDefaulter");
 const faunadb = require("faunadb"),
-    q = faunadb.query;
+    fq = faunadb.query;
+const lodash = require("lodash");
+const { getGeocode } = require("./../getGeocode");
 
-var client = new faunadb.Client({ secret: process.env.FAUNA_DB_DEV });
+DEBUG = false; // set to false to disable debugging
+function debug_log() {
+    if (DEBUG) {
+        console.log.apply(this, arguments);
```

Comment:
I might be going crazy but where is `arguments` coming from?

Reply:
I grabbed this solution from here: https://javascript.plainenglish.io/lets-extend-console-log-8641bda035c3
but I'm refactoring to make this more intuitive.